azure-devops/projects/pagopa-packages-projects/secrets.tf

```diff
@@ -12,5 +12,7 @@ module "secrets" {
     "io-azure-devops-github-EMAIL",
     "io-azure-devops-github-USERNAME",
     "pagopa-npm-bot-TOKEN",
+    "JIRA-USERNAME",
```

Comment:
can't we use an API token instead?

Reply:
Yes, in JIRA_PASSWORD must be entered the token. I used the environment variable name JIRA_PASSWORD and not JIRA_TOKEN because it was the same environment variable used in io-app. But I believe it can be changed

Reply:
I suggest to add a `DANGER-` prefix so the secrets names should be `DANGER-JIRA-USERNAME` , `DANGER-JIRA-TOKEN`.

Reply:
Done!